core-api/src/main/java/com/optimizely/ab/config/parser/JsonConfigParser.java

```diff
@@ -306,9 +304,8 @@ public ProjectConfig parseProjectConfig(@Nonnull String json) throws ConfigParse
             String id = audienceObject.getString("id");
             String key = audienceObject.getString("name");
             Object conditionsObject = audienceObject.get("conditions");
-            JSONArray conditionJson = (JSONArray)conditionsObject;
 
-            Condition conditions = ConditionUtils.<UserAttribute>parseConditions(UserAttribute.class, conditionJson);
+            Condition conditions = ConditionUtils.<UserAttribute>parseConditions(UserAttribute.class, conditionsObject);
```

Comment:
Nice! I can't comment in `parseAudiences`, but don't we need to update it too? We want it to pass `conditionsObject` straight to `ConditionUtils.parseConditions` without converting to `JSONArray conditionJson`.

Reply:
No audience.conditions is legacy and should never change.  This change is for typedAudiences and Experiment.audienceConditions as mentioned in the ticket.